package.json

```diff
@@ -6,9 +6,11 @@
   "scripts": {
     "build": "npm run lint:md && npm run lint:code && node ./src/build.js",
     "test": "npm run test",
+    "lint": "./node_modules/.bin/markdownlint '**/*.md' && ./node_modules/.bin/eslint . && eslint --ext js,md",
```

Comment:
@m-murasovs , can't you make this simply:

```
"lint": "npm run lint:code && npm run lint:md"
```

and the same for "lint:fix" to get a rid of that duplicate code?

Reply:
Ah, I'm not that smart!